src/lib/backend.ai-client-node.js

```diff
@@ -3168,14 +3168,14 @@ class ScalingGroup {
      * @param {string} description - Scaling group description
      * @param {string} wsproxyAddress - wsproxy url (NEW in manager 21.09)
      */
-    async create(name, description = "", wsproxyAddress = null) {
+    async create(name, description = "", scheduler_opts = "{}", wsproxyAddress = null) {
```

Comment:
The "order" of parameter should be kept as original (for API compatibility / legacy support). scheduler_opts will be the end of parameter.